candid/src/parser/value.rs

```diff
@@ -319,6 +319,78 @@ impl IDLValue {
     }
 }
 
+pub mod pretty {
+    use super::*;
+    use crate::pretty::*;
+
+    use ::pretty::RcDoc;
+
+    pub fn pp_label(id: &Label) -> RcDoc {
+        match id {
+            Label::Named(id) => str("\"")
```

Comment:
You may reuse code here to quote field names only when necessary: https://github.com/dfinity/candid/pull/62/commits/a7df6abddd52f9c23355add965c1279c1b1b3764

Reply:
Okay, I duplicated the code because those helpers are all private.  I realize that's not the best solution.

For now, I will make the one I need public (`pp_label`) and re-export it from where I am defining the duplicate above.

Maybe consider making the rest public, in another PR?